library.rb

```diff
@@ -1,3 +1,120 @@
+require './author'
+require './book'
+require './reader'
+require './order'
+
 class Library
+  attr_reader :authors, :books, :readers, :orders
+  def initialize
+    @authors = []
+    @books = []
+    @readers = []
+    @orders = []
+    puts 'Use :load method for load library from file.'
+  end
+
+  def load
+    load_from_file('Author', @authors, 'author.txt')
+    load_from_file('Book', @books, 'book.txt')
+    load_from_file('Reader', @readers, 'reader.txt')
+    load_from_file('Order', @orders, 'order.txt')
```

Comment:
1. we can use only string author, book... as parameter and produce class name, instance variable name and file name using it
2. we can use array of 'author', 'book' ... and call load_from_file in the loop